src/services/Discord.js

```diff
@@ -41,11 +43,16 @@ const splitMessageString = (msgString) => {
 *
 * @param {Object} channel
 * @param {String} message
+* @param {String} fileUrl
 */
-const sendMessageToChannel = (channel, message) => {
+const sendMessageToChannel = (channel, message, fileUrl = undefined) => {
```

Comment:
No need explicitly default it to `undefined`. It will implicitly be `undefined` if no variable is provided.

Reply:
Fixed in https://github.com/vostpt/bot/pull/154;